packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -345,11 +355,56 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     spotlightedParticipants
   ]);
 
+  /* @conditional-compile-remove(media-access) */
+  const meetingMediaAccess = useSelector(getMediaAccessSetting);
+  /* @conditional-compile-remove(media-access) */
+  const onToggleParticipantMicPeoplePaneProps = useMemo(() => {
+    return {
+      onForbidAudio: ['Unknown', 'Organizer', 'Presenter', 'Co-organizer'].includes(role ?? '')
```

Comment:
How come you not using capabilities here like so?
`onForbidAudio: capabilities?.forbidOthersAudio?.isPresent ? onForbidAudio : undefined,`

Reply:
this might be cleaner with capabilities I think thats a good idea to look into if its available

Reply:
Fixed. Thanks